### ISSUE TITLE
Requesting *.scala files causes errors in 3.2.1

### ISSUE THREAD

**Issue by aivarannamaa** [jcrodriguez-dis/moodle-mod_vpl#38]
If you request a *.scala file, eg. _HelloWorld.scala_ with following content:

```
  object HelloWorld {
    def main(args: Array[String]) {
      println("Hello, world!")
    }
  }
```
then main page of the VPL activity shows only first word of the requested file and throws an error:

```
Exception - Undefined class constant 'endTag'
Stack trace:
    line 34 of \mod\vpl\views\sh_scala.class.php: Error thrown
    line 265 of \mod\vpl\views\sh_scala.class.php: call to vpl_sh_scala->show_pending()
    line 364 of \mod\vpl\filegroup.class.php: call to vpl_sh_scala->print_file()
    line 67 of \mod\vpl\view.php: call to file_group_process->print_files()
```

If I rename the requested file to eg. _HelloWorld.cala_ then everything works again.

**Comment by aivarannamaa** [comment]
Looks like the solution is to remove `echo self::endTag;` on line 34 in \mod\vpl\views\sh_scala.class.php

**Comment by jcrodriguez-dis** [comment]
Hello Aivar,
Yes, it is correct the problem is in the on line 34 in \mod\vpl\views\sh_scala.class.php , but the fix isto  change "endTag" by "ENDTAG". 
This bug affect to other syntax highlighters: Ada, Pascal, and Fortran.

It has been resolved in V3.2.4

**Comment by aivarannamaa** [comment]
Seems that branch v3.2.4 is at the moment equal to v3.3

Will the next version be 3.3 or 3.2.4? Is it OK to install current state of branch 3.2.4 (or 3.3) as version 3.3 or will it cause trouble when eg. 3.2.5 comes out?

**Comment by jcrodriguez-dis** [comment]
Oops, they must not be equals. Please, don't install v3.3 it is a development version. Apologize for this error, this is due my pour skills with Git. I tryed to pull the fixes in v3.2.4 to v3.3 and it seems that I did the inverse.
I will resolve the problem.

**Comment by jcrodriguez-dis** [comment]
I have recreate the v3.2.4 with the same original fixes.
Thanks Aivar.
Best regards,
Juan Carlos
